README.md

```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
+   ```
 
+   ```
+   kubectl cp odim/redis-ha-ondisk-primary-0:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
+
+   **For example(in-memory)**:
+   
+   ```
+kubectl cp odim/redis-ha-inmemory-primary-0:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
+   
+   ```
+kubectl cp odim/redis-ha-inmemory-primary-0:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
+   
+   These are the backup files to be restored.
+
+4. Copy the above backup files to the pod to restore them.
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/<pod-name>:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/pod-name>:/redis-data/appendonly.aof.old
+   ```
+
+   **For example**:
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/redis-ha-ondisk-primary-0:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/redis-ha-ondisk-primary-0/redis-data/appendonly.aof.old
+   ```
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/redis-ha-inmemory-primary-0:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/redis-ha-inmemory-primary-0/redis-data/appendonly.aof.old
+   ```
+
+5. Log in to redis CLI and disable append-only configuration to restore the data back.
+
+   ```
+   redis-cli --tls --cert /etc/odimra_certs/odimra_server.crt --key /etc/odimra_certs/odimra_server.key --cacert /etc/odimra_certs/rootCA.crt -a <password>
+   ```
+
+   ```
+   CONFIG SET "appendonly" no
+   ```
 
+6. Delete the data available to verify later, if the restore option works (optional).
+
+   ```
+   keys *
+   ```
+
+   ```
+   FLUSHALL
+   ```
+
+7. Exit from the Redis CLI and move to the redis-data directory.
+
+8. Remove the existing rdb and aof files and replace the backed up ones with the name.
+
+   ```
+   cd /redis-data
+   ```
+
+   ```
+   rm -rf dump.rdb appendonly.aof
+   ```
+
+   ```
+   mv dump.rdb-1 dumb.rdb
+   ```
+
+   ```
+   mv appendonly.aof.old appendonly.aof
+   ```
+
+9. Get the pods of the Redis (either in-memory or on-disk) and restart  them.
+
+   ```
+   kubectl get pods -nodim | grep redis | grep inmemory
+   ```
+
+   ```
+   kubectl get pods -nodim | grep redis | grep ondisk
+   ```
+
+   ```
+   kubectl delete pods <pod1> <pod2>..<pod3> -nodim
+   ```
+
+   **For example**:
+
+   ```
+   kubectl delete pods redis-ha-inmemory-primary-0 redis-ha-inmemory-secondary-0 redis-ha-inmemory-secondary-1 -nodim
+   ```
+
+10. Once all the pods have started and are in running state, get inside the pod, log in to Redis CLI and check for the old data.
+
+    ```
+    kubectl exec -it redis-ha-ondisk-primary-0 -nodim bash
+    ```
+    ```
+    kubectl exec -it redis-ha-inmemory-primary-0 -nodim bash
+    ```
+    ```
+    redis-cli --tls --cert /etc/odimra_certs/odimra_server.crt --key /etc/odimra_certs/odimra_server.key --cacert /etc/odimra_certs/rootCA.crt -a <password>
+    ```
+
```

Comment:
same as above comment. from where redis-cli has to be executed and what is the command to get into the pod

Reply:
fixed